opencog/query/Implicator.cc

```diff
@@ -49,8 +49,12 @@ bool Implicator::grounding(const HandleMap &var_soln,
 	if (_result_set.size() >= max_results)
 		return true;
 
-	Handle h = inst.instantiate(implicand, var_soln);
-	insert_result(h);
+	// By wrapping this in a try/catch we can ignore the cases when
+	// executions create ill-formed atoms.
+	try {
+		Handle h = inst.instantiate(implicand, var_soln);
+		insert_result(h);
+	} catch(const InvalidParamException& e) {}
```

Comment:
All I'm saying is that the exception will clutter up the opencog log file with megabytes of crap.  Whatever it is that is throwing that exception should probably be changed to throw a non-logging exception.

I ca do this, but I am still behind, and haven't finished my last project, yet.